README.md

```diff
@@ -280,10 +280,10 @@ You have successfully canceled your booking
 - `airline_backend_system` is the system that processes messages sent through the Ballerina message broker. You will see the following logs printed on your console when you run `guide.flight_booking_system`
 
 ```
-2018-04-23 21:08:09,475 INFO  [airline_backend_system] - [NEW BOOKING] Details :\
+2018-04-23 21:08:09,475 INFO [{PACKAGE_NAME}/airline_backend_system:{VERSION}] - [NEW BOOKING] Details :\
```

Comment:
`PACKAGE_NAME` should actually be the org name here right? and why isn't the actual org name used here, given that this is an actual log output